pegasus/sites.v3/code.org/public/educate/wa-districts.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: District partnerships
 nav: educate_nav
+theme: responsive
 ---
 # Partner with Code.org to bring K-12 computer science to your district
 
```

Comment:
clicking on "See all 90 partner districts" leads to /educate/partner/district-partners, which has a map that doesn't quite scale to small screens